src/openrct2/peep/Staff.cpp

```diff
@@ -643,7 +643,10 @@ bool Staff::DoHandymanPathFinding()
                 bool chooseRandom = true;
                 if (litterDirection != INVALID_DIRECTION && pathDirections & (1 << litterDirection))
                 {
-                    if ((scenario_rand() & 0xFFFF) >= 0x1999)
+                    /// When in a queue path make the probability of following litter much lower
+                    /// as handymen often get stuck when there is litter on a normal path next to a queue they are in
+                    uint32_t chooseRandomProbability = pathElement->IsQueue() ? (0xFFFF - 0x1999) : 0x1999;
```

Comment:
I think it would be better to check if it is a queue and it has a ride assigned to it. That way if someone has used a queue path for cosmetic reasons it would not be affected.

Reply:
I added a check with `GetRideIndex`